packages/editor/src/client/base.tsx

```diff
@@ -189,6 +190,13 @@ export function Base({children}: BaseProps) {
                   <DropdownItemLink route={LogoutRoute.create({})}>
                     {t('navbar.logout')}
                   </DropdownItemLink>
+                  <TourMethods>
```

Comment:
We also need the possibility to start different tours. If you never started the editor you get the basic introduction tour but if we implemented a new feature we'll want to start that one. I think there should be some options inside shepherd to control this.